src/core/lavfindexer.cpp

```diff
@@ -189,6 +205,13 @@ FFMS_Index *FFLAVFIndexer::DoIndexing() {
 	}
 
 	TrackIndices->Finalize(VideoContexts);
+
+	for (unsigned int i = 0; i < FormatContext->nb_streams; i++) {
+		if (VideoContexts[i].CodecContext != nullptr) {
+			avcodec_close(VideoContexts[i].CodecContext);
```

Comment:
SharedVideoContext's destructor closes the CodecContext, so that should instead be extended to also free the context.